<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Керування картами</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>410</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Відкрити нову карту</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="distributeButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Розподілення</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>701</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Каталог карт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addMapToDBButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>410</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Додати карту </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметри карти:</string>
    </property>
   </widget>
   <widget class="QLabel" name="filenameLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="removeMapButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Видалити карту</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SortButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортування за маштабом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAllMaps">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>410</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Додадти каталог</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanFolder">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>30</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистити каталог</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
